pkg/internal/checkup/executor/testpmd-console.go

```diff
@@ -38,12 +48,25 @@ const (
 
 const testpmdPrompt = "testpmd> "
 
-func (e Executor) runTestpmd(vmiName string) error {
+func NewTestpmdConsole(testpmdClient VmiSerialConsoleClient, namespace, vmiEastNICPCIAddress,
+	vmiEastEthPeerMACAddress, vmiWestNICPCIAddress, vmiWestEthPeerMACAddress string, verbosePrintsEnabled bool) *TestpmdConsole {
+	return &TestpmdConsole{
+		testpmdClient:            testpmdClient,
+		namespace:                namespace,
+		vmiEastEthPeerMACAddress: vmiEastEthPeerMACAddress,
+		vmiWestEthPeerMACAddress: vmiWestEthPeerMACAddress,
+		vmiEastNICPCIAddress:     vmiEastNICPCIAddress,
+		vmiWestNICPCIAddress:     vmiWestNICPCIAddress,
+		verbosePrintsEnabled:     verbosePrintsEnabled,
+	}
+}
+
+func (t *TestpmdConsole) runTestpmd(vmiName string) error {
```

Comment:
AI - the object is not changing anything inside it, so it should not use a pointer to it.

Reply:
DONE